bin/deepspeed

```diff
@@ -1 +1 @@
-ds
\ No newline at end of file
+ds
```

Comment:
I am sure you didn't modify these 4 files. Can you please remove them from the PR?